src/util/common.js

```diff
@@ -63,3 +63,17 @@ export const _procError = err => {
 export function isAuth() {
   return !!localStorage.getItem('auth-token')
 }
+
+export function getUserType() {
+  let token = localStorage.getItem('auth-token')
+  let userType = token.substr(token.length - 1)
+  switch (userType) {
```

Comment:
how about just: return userType  ?

Reply:
ok, I will only return last char, checking char will be handled by router